File: src/SwordToOffer/minNumRotate.js
function minNumberInRotateArray(rotateArray) {
    let length = rotateArray.length;
    if (length === 0) {
        return 0;
    }
    let min = rotateArray[0];
    for (let i = 0; i < length; i++) {
        if (min > rotateArray[i]) {
            min = rotateArray[i];
        }
    }
    return min;
    // write code here
}

File: src/SwordToOffer/minNumRotate.java
package SwordToOffer;

import java.util.ArrayList;

public class minNumRotate {
    public int minNumberInRotateArray(int[] array) {
        if (array.length == 0) {
            return 0;
        }
        int length = array.length;
        int min = array[0];
        for (int i = 0; i < length; i++) {
            if (min > array[i]) {
                min = array[i];
            }
        }
        return min;
    }
}